mavproxy/PKGBUILD

```diff
@@ -1,16 +1,16 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 
 pkgname=mavproxy
-pkgver=1.8.34
+pkgver=1.8.35
 pkgrel=1
 pkgdesc='MAVLink proxy and command line ground station.'
 arch=('any')
 url='http://ardupilot.github.io/MAVProxy/html/index.html'
 license=('GPL3')
-depends=(python python-pymavlink)
+depends=(python python-pymavlink python-opencv)
```

Comment:
`python-opencv` is not a package anymore. `opencv` provides the python bindings in Archlinux

Reply:
According to upstream mavproxy setup.py, `python-opencv` is not a dep anyway to the package.